test_wazuh/test_fim/test_recursion_level/test_recursion_level.py

```diff
@@ -91,30 +81,23 @@ def recursion_test(dirname, subdirname, recursion_level, timeout=1, threshold_tr
             if ((recursion_level < threshold_true * 2) or
                 (recursion_level >= threshold_true * 2 and n < threshold_true) or
                 (recursion_level >= threshold_true * 2 and n > recursion_level - threshold_true)):
-                check_event_too_long()
                 regular_file_cud(path, wazuh_log_monitor, time_travel=is_scheduled, min_timeout=timeout)
 
         # Check False (exceding the specified recursion_level)
         for n in range(recursion_level, recursion_level + threshold_false):
-            check_event_too_long()
             path = os.path.join(path, subdirname + str(n + 1))
             regular_file_cud(path, wazuh_log_monitor, time_travel=is_scheduled, min_timeout=timeout, triggers_event=False)
 
-    except (TimeoutError, AssertionError):
-        check_event_too_long()
-
-
-def check_event_too_long():
-    """Checks if the `Event to long` message was raised due to Whodata path length limitation."""
-    try:
-        event = wazuh_log_monitor.start(timeout=1,
-                                        callback=callback_audit_event_too_long,
-                                        update_position=False).result()
-        if event:
-            pytest.exit(returncode=pytest.ExitCode.OK)
-        pytest.fail("No Event Too Long message was raised.")
     except TimeoutError:
-        pass
+        if wazuh_log_monitor.start(timeout=2, callback=callback_audit_event_too_long, update_position=False).result():
```

Comment:
DEFAULT_TIMEOUT ??

Reply:
Similar to the previous change requested. We are not using the DEFAULT_TIMEOUT value because we need a different value in this case.